helm/credentiald-chart/templates/networkpolicy.yaml

```diff
@@ -23,12 +23,18 @@ spec:
     - namespaceSelector:
         matchLabels:
           name: kube-system
-  # API access
+  # API access - we have to whitelist all private IP ranges
+  # as the apiserver is using host networking.
   - ports:
     - port: 443
       protocol: TCP
     to:
-    - namespaceSelector: {}
+    - ipBlock:
+        cidr: 10.0.0.0/8
+    - ipBlock:
+        cidr: 172.16.0.0/12
+    - ipBlock:
+        cidr: 192.168.0.0/16
```

Comment:
As a general suggestion for network policies: Is it possible to comment the policies to express what they suppose to mean? E. g. "Allow traffic to the K8s API"

Reply:
Ignore me. The comment in lines 26-27 is doing just that.